apps/google-analytics-4/frontend/src/components/config-screen/HomeAnalyticsPage.tsx

```diff
@@ -0,0 +1,218 @@
+import { useCallback, useState, useEffect } from 'react';
+import { AppExtensionSDK } from '@contentful/app-sdk';
+import { Box } from '@contentful/f36-components';
+import { css } from 'emotion';
+import omitBy from 'lodash/omitBy';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import tokens from '@contentful/f36-tokens';
+import ApiAccessPage from './api-access/ApiAccessPage';
+import {
+  convertServiceAccountKeyToServiceAccountKeyId,
+  convertKeyFileToServiceAccountKey,
+  AssertionError,
+} from '../../utils/serviceAccountKey';
+import type { AppInstallationParameters, ServiceAccountKey, ServiceAccountKeyId } from '../../types';
+import AboutSection from './header/AboutSection';
+import ConfigurationPage from './configuration/ConfigurationPage';
+import Splitter from './utils/Splitter';
+import GoogleAnalyticsIcon, { googleAnalyticsBrand } from './utils/GoogleAnalyticsIcon';
+
+const styles = {
+  body: css({
+    height: 'auto',
+    minHeight: '65vh',
+    margin: '0 auto',
+    marginTop: tokens.spacingXl,
+    padding: `${tokens.spacingXl} ${tokens.spacing2Xl}`,
+    maxWidth: tokens.contentWidthText,
+    backgroundColor: tokens.colorWhite,
+    zIndex: 2,
+    boxShadow: '0px 0px 20px rgba(0, 0, 0, 0.1)',
+    borderRadius: '2px',
+  }),
+  background: css({
+    display: 'block',
+    position: 'absolute',
+    zIndex: -1,
+    top: 0,
+    width: '100%',
+    height: '300px',
+    backgroundColor: googleAnalyticsBrand.primaryColor,
+  }),
+};
```

Comment:
what are your thoughts on having a separate file for additional styles? Such as `HomeAnalyticsPage.styles.ts` ... or something like that. I have found in the past this helps with separation of concerns and file readability. And on the same topic, thoughts on a overarching directory for `HomeAnalyticsPage` that contains all related files (component, testing, styles, mocks etc) ?

Reply:
I'm personally a fan of the styles idea.

Also I totally forgot to mention this in the description. I wanted to mention for a given app component/IFrame, we treat the locations components (ie. ConfigScreen, Sidebar etc...) as the App.tsx for that component and then build out the file component file directories based on the location in the `components` directory. So as for your comment I think it makes sense to create a separate directory like `components/index` to have all the HomeAnalyticsPage and other landing pages for those components to live there. Hopefully I understood what you were getting at.